### ISSUE TITLE
Some text is missing in Paprika theme when opening email in the Builder

### ISSUE THREAD

**Issue by LordRembo** [mautic/mautic#13618]
### Mautic Version

5.0.x series

### Way of installing

I cloned the Mautic repo (not recommended or supported)

### PHP version

8.2

### What browsers are you seeing the problem on?

Not relevant

### What happened?

When trying to make an new email using the Paprika theme, there is text missing underneath the big green titles, when compared to the saved email. (originally tested and reported by @code5rick on another issue)

![Screenshot 2024-04-09 at 16 05 44](https://github.com/mautic/mautic/assets/758957/f688eb26-5070-446f-9be7-ca491fc70d4b)

### How can we reproduce this issue?

Step 1: Make a new Email
Step 2: select Paprika theme
Step 3: open the Builder
Step 4: see that there is no green text underneath the 2 Text blocks which have green, uppercase titles 
Step 5: close Builder and hit 'Save and close'
Step 6: open the Email preview
Step 7: see that those same blocks now do contain smaller, green, lorem ipsum text

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>
